app/components/Form/ObjectPermissions.tsx

```diff
@@ -24,12 +25,14 @@ const ObjectPermissions = ({
   canEditGroups,
   canViewGroups,
   requireAuth,
+  authors,
   ...props
 }: {
-  canEditUsers?: any;
-  canEditGroups?: any;
-  canViewGroups?: any;
-  requireAuth?: any;
+  canEditUsers?: PublicGroup[];
+  canEditGroups?: PublicGroup[];
+  canViewGroups?: PublicGroup[];
+  requireAuth?: boolean;
+  authors?: number[];
```

Comment:
Hmm, I realize now that this is all in ObjectPermission related stuff.

I don't know how fan I am of this, since authors don't really have anything to do with ObjectPermissions and is specific to just articles.